### PR TITLE
added DPG to media

### PR DESCRIPTION
**Pull request by KennyPaul**

### CODE REVIEW

File: content/media/dpg.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+![Tazama+DPG Logo](https://www.linuxfoundation.org/hs-fs/hubfs/Tazama%20image.jpg?width=557&height=166&name=Tazama%20image.jpg)
```
Comment (hythloda) [inline content/media/dpg.md:2]: Did the image render for you? 
Screenshot 2024-10-04 at 6 26 23 PM
The URL has query parameters and those sometimes interfere with image rendering in markdown. We might want to host it locally. Although then we loose hs clicks

File: content/media/media.md [hunk 2]
```diff
@@ -14,15 +14,17 @@ title: "Contact"
 
 ## Recent Media
 
-- [Linux Foundation Launches Tazama: A Revolutionary Open Source Solution for Real-Time Fraud Management](https://www.linuxfoundation.org/press/linux-foundation-launches-tazama-for-real-time-fraud-management)
+- 9 September 2024 [Tazama is Recognitized as a Digital Public Good](https://tazama.org/media/dpg.md)
```
Comment (hythloda) [inline content/media/media.md:17]:
```suggestion
- 9 September 2024 [Tazama is Recognitized as a Digital Public Good](../dpg)
```
Locally hosted files need relative path and not md.

File: content/media/media.md [hunk 3]
```diff
@@ -14,15 +14,17 @@ title: "Contact"
...
 
-- [How a new fraud detection software can help millions more people bank safely](https://www.gatesfoundation.org/ideas/articles/open-source-financial-inclusion-tazama)
+- 11 March 24 [Anti-Fraud Project Boosts Security of African, Asian Financial Systems](https://www.darkreading.com/cyber-risk/anti-fraud-project-boosts-security-of-african-asian-financial-systems)
```
Comment (hythloda) [inline content/media/media.md:19]: Love the date additions!

File: content/media/dpg.md (hunk 1 above)
Comment (hythloda) [reply content/media/dpg.md:2]: oh, and I think the website is set up for png.

Comment (hythloda) [inline content/media/dpg.md:2]:
```suggestion
![Tazama+DPG Logo](/image/Tazama+DPG.jpg)
```

**Review by hythloda — COMMENTED** [review]
Also need to change the jpg to png

File: content/media/dpg.md (hunk 1 above)
Comment (KennyPaul) [reply content/media/dpg.md:2]: well it rendered happily in the markup checker version at least.
I have a version of the logo in the PR, (albiet jpg), in the static/image folder, but I wasn't completely sure of the actual path to stick into the md file so for the time being I had just added the link to the LF web location.

**Review by hythloda — APPROVED** [review]
Loads great now for me too!

**Pull request merged**